firmware/src/rdm.h

```diff
@@ -122,6 +122,26 @@ enum {
 };
 
 /**
+ * @brief The recording mask for the recorded value support bit field.
+ */
+enum { SENSOR_SUPPORTS_RECORDING_MASK = 0x01 };
+
+/**
+ * @brief The lowest / highest mask for the recorded value support bit field.
+ */
+enum { SENSOR_SUPPORTS_LOWEST_HIGHEST_MASK = 0x02 };
+
+/**
+ * @brief The all-sensors value.
+ */
+static const uint8_t ALL_SENSORS = 0xff;
+
+/**
+ * @brief Indicates that recorded / highest / lowest values aren't supported.
+ */
+static const uint16_t SENSOR_VALUE_UNSUPPORTED = 0;
```

Comment:
Okay. I assumed the compiler would just swap it out so there wouldn't be a memory impact.